content/1.7/back-office/dashboard/dashboard-view.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Dashboard view
+weight: 1
+---
+
+# Dashboard view
+
+Dashboard is the first default page on the back office, in this page the shop performance is displayed.
+
```

Comment:
The dashboard is divided into 4 main areas: a horizontal bar, and three columns.